src/aics-image-viewer/components/App/index.tsx

```diff
@@ -719,6 +728,8 @@ export default class App extends React.Component<AppProps, AppState> {
     view3d.setCameraMode(enums.viewMode.VIEW_MODE_ENUM_TO_LABEL_MAP.get(userSelections.mode));
     view3d.setShowAxis(userSelections[SHOW_AXES]);
     view3d.setShowBoundingBox(aimg, userSelections.showBoundingBox);
+    view3d.setBackgroundColor(colorArrayToFloatArray(userSelections.backgroundColor));
+    view3d.setBoundingBoxColor(colorArrayToFloatArray(userSelections.boundingBoxColor));
```

Comment:
setBoundingBoxColor needs to pass in aimg ?

Reply:
looks like just about all the setters in volume-viewer with a `volume` argument (the ones that require `aimg`) leave that argument unused and instead check `this.image`, potentially to future-proof against the viewer eventually supporting multiple images at a time? if so, `setBoundingBoxColor` will need to receive a volume to know which of multiple bounding boxes to recolor.

Reply:
yes, setBoundingBoxColor should get an aimg passed in for sure.